Tests/XcodeGenKitTests/SourceGeneratorTests.swift

```diff
@@ -9,7 +9,11 @@ import TestSupport
 
 class SourceGeneratorTests: XCTestCase {
 
-    func testSourceGenerator() {
+    func testSourceGenerator() throws {
+        #if os(Linux) && swift(<6.0.2)
```

Comment:
Swift version is constrained to ensure it tries to run again when a new release is out.